.eslintrc.js

```diff
@@ -0,0 +1,24 @@
+module.exports = {
```

Comment:
Are these just the default settings? Just want to understand better

Reply:
some are default but most are there to make eslint conform to our standard. For example Eslint by default will use 4 spaces, not 2 and gets upset with a bunch of stuff like `console.log`. These settings basically make it pass the same stuff that `prettier` finds as well as catches the stuff it misses.

Reply:
Just looking at the docs now there are a ton of rules we might want to add, any recommendations on what is minimum viable subset to include or is this good for now 

https://eslint.org/docs/rules/

Reply:
so most of these are enabled by default. The ones I explicitly added to this config are exclusions that are ignored because it conflicts with our prettier config. For now I think this is good enough and we can modify as we need later on.